tests/components/history_stats/test_sensor.py

```diff
@@ -118,7 +118,7 @@ def test_period_parsing(self, mock):
         assert sensor2_end.minute == 0
         assert sensor2_end.second == 0
 
-    def test_measure(self):
+    async def async_test_measure(self):
```

Comment:
I didn't know that this was supported. Did you check that the changed tests were actually awaited?

Reply:
Will check to be sure.

Reply:
Nope you are right, not actually awaited 🤦   The manual testing looked good, and I didn't catch the test not being run.

I moved them out.